client/scripts/views/pages/new_thread.ts

```diff
@@ -174,15 +192,17 @@ export const NewThreadForm: m.Component<{}, IState> = {
             oncreateBind: (state) => {
               vnode.state.quillEditorState = state;
             },
+            editorNamespace: 'new-discussion',
             tabindex: 2,
-            editorNamespace: 'new-link',
           }),
         ]),
         m(FormGroup, [
           m(AutoCompleteTagForm, {
```

Comment:
Reused again here

Reply:
I mean, I could set the component to default to sets of tags and featuredTags if nothing's passed in, which would reduce redundant passing of attrs, but the passing of state needs to be reinstantiated for each use, yeah?